### PR TITLE
Make course explorer full width

### PR DESCRIPTION
**Pull request by caleybrock**
before:
screen shot 2017-09-27 at 4 36 41 pm

after:
screen shot 2017-09-27 at 4 33 50 pm

Makes the course explorer width work with 970 page widths on the rest of /courses.
Technically it only gets to 968, but I didn't want to split the pixels too much and I think this looks good enough. 
Approved by @tanyaparker 
cc @breville

### CODE REVIEW

**Comment by caleybrock** [comment]
Feel free to merge this @Erin007 whenever it passes tests, and if you're happy with it. I'll be on vacation :)

File: shared/haml/course_explorer_table.haml [hunk 1]
```diff
@@ -6,7 +6,7 @@
   :css
     .courses .bar-#{n} {
       float: left;
-      width: #{n * 72}px;
+      width: #{n * 74.5}px;
```
Comment (breville) [inline shared/haml/course_explorer_table.haml:9]: Interesting.  Does the subpixel rendering become obvious anywhere?

**Comment by breville** [comment]
I'd make sure this looks okay on the pegasus version at code.org/educate/curriculum/courses, and also that the responsive variations aren't affected.

**Comment by caleybrock** [comment]
Responsive versions have been tested, and so has the code.org/educate/curriculum/courses page.
screen shot 2017-09-27 at 4 54 19 pm

File: shared/haml/course_explorer_table.haml (hunk 1 above)
Comment (caleybrock) [reply shared/haml/course_explorer_table.haml:9]: My understanding is that subpixel rending is a problem in like IE 5/6, but not modern browsers we support.

Comment (breville) [reply shared/haml/course_explorer_table.haml:9]: Cool, just wanted to make sure there weren't any visual artefacts because of the subpixel values.

**Pull request merged**